.drone.yml

```diff
@@ -32,6 +34,44 @@ steps:
       event: push
       branch: main
 
+
+---
+kind: pipeline
+type: kubernetes
+name: callisto-ingress-test-deploy
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+  - name: lint helm files
+    image: pelotech/drone-helm3
+    settings:
+      mode: lint
+      chart: ./helm
+      values_files:
+        - ./helm/test-values.yaml
+
+  - name: deploy to test
+    image: pelotech/drone-helm3
+    settings:
+      namespace: callisto-test
+      mode: upgrade
+      chart: ./helm
+      values_files:
+        - ./helm/test-values.yaml
+      release: callisto-ingress-nginx
+      kube_certificate: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUMyRENDQWNDZ0F3SUJBZ0lSQUxBdk1yVGVzQkhNUkliR0ZZa3JsaHN3RFFZSktvWklodmNOQVFFTEJRQXcKRlRFVE1CRUdBMVVFQXhNS2EzVmlaWEp1WlhSbGN6QWVGdzB4TnpBNE1qZ3hPVFU0TlRSYUZ3MHlOekE0TWpneApPVFU0TlRSYU1CVXhFekFSQmdOVkJBTVRDbXQxWW1WeWJtVjBaWE13Z2dFaU1BMEdDU3FHU0liM0RRRUJBUVVBCkE0SUJEd0F3Z2dFS0FvSUJBUURQNm5ZVGpmbmR2T2kxOEltYm5idk9aOXJ3c1hxSkZFTE1IVTkxT2VIaXFIT2kKT1luVFZ3cmpCaDJpZExUM0ozbkVoYnM0WFRIcVlwcVEzdmcwVzluVVRPMnk5ZmFmdlRZVTk0NGZXUTl1SXJOZwo1Y0Q0SXhzK05vWTJmdkkrcStYUUM2SFNiRGJ0S0dOUy9Kbmc5SFdsazF2dzY0TlBwbU5IMTJtUFVvNGpGZnc5CmZoL3RwY1k5akVsZnhJckJBVEJzNHhYcDJHOFFrK1ZCSWE5bnF4bmtVTHJvT0t2Y092T3g4RDU1TXdXcEVKc3QKdENZSXFHM05xdlhMcUdTQzdueHoxYmJNSmZjRDdBRmdvZnM1bXZ0bmlRTHBnZG93WmxNc3g2cnJVM2pBUkw3OApvTktXR3p3c3BKNjhybXdhSGZ4NlRBWldHbUxSU2NPd0xlbWJ3cDRIQWdNQkFBR2pJekFoTUE0R0ExVWREd0VCCi93UUVBd0lCQmpBUEJnTlZIUk1CQWY4RUJUQURBUUgvTUEwR0NTcUdTSWIzRFFFQkN3VUFBNElCQVFDb2RrZkMKS2NOYzY5cXR6ZllWUUFlajFVb0hhMjhUUzlqMHV3RVdqWlU0dFVWR2cvZDNCK2YvcnYyN2tqRGM3U2JJSForVQpxMlpMVlVnYU85OVoyUjQyUFhNVTZCRUVUdjUvT0t4amdkNlFNQnFFVkxBdU1XZFpsbnBkMEpDMDFGd0tJb3BGCnVpVUFoWnRjR0tJRXg1U1N1SUlKdFU4a2pnR2xuS0V6aXJrUVVCOUJUOURSQ1VhelQ1ZUp6VFVLSVliWjBNL3gKSUZMbUQvZmZZeFhmRGlSM25YWkdZSWpkUGJHYW42ZU5pZExjQkdEZkxabzZFdkJTR2FUSHl5KzEyc3c2clhTVwpBTVFWd3BUY2ZpSlM1NG9xSXhDU3dyam5XNFZYRnhiSC96SWZXMGs5M29ucERNdXhjT20rN0NuZjViRlJEdlpzCjRCaEV1ank1eEVtajA1TWwKLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo=
```

Comment:
I know we haven't for dev. But is it worth putting this certificate in a secret?

Reply:
Why are we putting it in a secret it's a public key

Reply:
@JWeston-HO was this suggestion based around this ticket:
https://collaboration.homeoffice.gov.uk/jira/browse/EAHW-2321
which I think was originally mentioned by @taomoh 

@stephenkaye-ho maybe we keep the public cert hardcoded here, and it can be reviewed as part of the above ticket?

Reply:
I wondered if this became a moot point because of a difference concern. Hardcoding it make it a bit more difficult should it ever need to change. Think of the repos we'd have to update. Therefore I was thinking a secret is good, not because it's sensitive but because it allows us to manage a very common setting more centrally. This is only true if we can add it at the "organisation" level though.

Reply:
Neither myself or Stephen are able to add the secret to the organisation. A conversation with ACP to get it into there is ongoing. So for now the cert is being stored in the repo secrets.
If they agree to it, we can have a separate PR to just change the secret reference in here to the org one.